helion/_compiler/device_ir.py

```diff
@@ -693,6 +693,25 @@ def visit_Attribute(self, node: ast.Attribute) -> object:
                 raise exc.CantReadOnDevice(type_info) from None
         return getattr(self.visit(node.value), node.attr)
 
+    def visit_Expr(self, node):
+        approved_ops = ["atomic_add"]
+
+        # Check if there is an inner call to an approved op
+        if isinstance(node.value, ast.Call):
+            func = node.value.func
+            op_name = None
+
+            if isinstance(func, ast.Name):
+                op_name = func.id
+
+            elif isinstance(func, ast.Attribute):
+                op_name = func.attr
+
+            if op_name in approved_ops:
+                return self.visit(node.value)
+
+        raise exc.StatementNotSupported(type(node).__name__)
```

Comment:
What ops are we trying to stop usage of?

Reply:
Removed